drone/src/agent/backend.rs

```diff
@@ -83,21 +83,11 @@ impl BackendMonitor {
         let backend_id = backend_id.clone();
 
         tokio::spawn(async move {
-            let container_name = backend_id.to_resource_name();
             tracing::info!(%backend_id, "Log recording loop started.");
-            let mut stream = docker.get_logs(&container_name);
+            let mut stream = docker.log_stream(&backend_id);
 
             while let Some(v) = stream.next().await {
-                match v {
-                    Ok(v) => {
-                        if let Some(message) = DroneLogMessage::from_log_message(&backend_id, &v) {
-                            nc.publish(&message).await?;
-                        }
-                    }
-                    Err(error) => {
-                        tracing::warn!(?error, "Error encountered forwarding log.");
```

Comment:
Did we lose this warning?

Reply:
It will result in “Log loop terminated” instead (the `log_stream` terminates if it encounters a parse error). My thinking is that we can add more logging if we start encountering that message.